config/manifests/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+resources:
+- ../default
+- ../samples
+- ../scorecard
```

Comment:
Where did `scorecard` come from?  I didn't have it [in my attempt at SDK v1.0.1][1].  And I don't see a `config/scorecard` in your tip:

```console
$ git ls-tree cde0dc77e7 -- config/
040000 tree ec4d134b363260017941b21a91707c809c363815    config/certmanager
040000 tree c5c218f132f09e55b2a1346c45a03961c3975373    config/crd
040000 tree 93494a3c9bdfd614b16c79e7cffe6bf5d4fb9ed6    config/default
040000 tree ebb074c993dfc68a5aa13c40221ae9d9b6c9aea6    config/manager
040000 tree 0b075d49b837e41817c0685134b3a7b3983c9232    config/manifests
040000 tree b535813978e2f2d3001990d684063a1d96bc9bed    config/olm-catalog
040000 tree ceedc8a2bee25b16deceb190b3ecfb02638ddfbc    config/prometheus
040000 tree 734fd7e3e4d2e67dc413916478fd062df5f21746    config/rbac
040000 tree ceece8f88259da3d69a81f5ffa8c82d5eb91d9c4    config/samples
040000 tree a878925b006459abe7238c43e5fe62c58d8174b2    config/webhook
```

[1]: https://github.com/openshift/cincinnati-operator/pull/79/files#diff-fe545782dbfdb2c00c396b25f7a8990e1d4af2d68d02e25363dc54c837f8aac6R3

Reply:
The `generate bundle` gets an error if it's not there. It gets created by v1.2 and I just copied it over to avoid the error.

Reply:
You're sure you don't have a `config/scorecard/` in your local checkout that just hasn't been committed?  I [had one][1].

[1]: https://github.com/openshift/cincinnati-operator/pull/79/files#diff-6aabd462c9a04e77fcde378dee12dda1b7988ac03c901e8eeb262bc03fbd6787R1

Reply:
Right. I purposely removed it but it's required by bundling so have added back.